Dockerfile

```diff
@@ -2,7 +2,7 @@
 FROM rust:1.25.0-stretch as builder
 ADD . /app
 WORKDIR /app
-ARG RUST_TOOLCHAIN=nightly-2018-04-29
+ARG RUST_TOOLCHAIN=nightly
```

Comment:
don't forget to keep this value in sync w/ .travis.yml's

Reply:
Thanks, missed that one .